Hardware/Hardware/CardReader/PaymentIntentParameters.swift

```diff
@@ -24,10 +24,6 @@ public struct PaymentIntentParameters {
     @StatementDescriptor
     public private(set) var statementDescription: String?
 
-    /// Email address that the receipt for the resulting payment will be sent to.
-    @Email
```

Comment:
If you remove this property, it should also be safe to remove a struct called Email in the hardware module, which is where the `@Email` annotation is declared (this is called a [property wrapper](https://docs.swift.org/swift-book/LanguageGuide/Properties.html)):
`public struct Email<Value: StringProtocol>`

Reply:
Good catch! Done - f48064cb64770dd5f5a904c6d393bab10789a663